src/foam/u2/view/RichChoiceView.js

```diff
@@ -259,10 +273,13 @@ foam.CLASS({
                   .select(section.filtered || section.dao, function(obj) {
                     return this.E()
                       .start(self.rowView, { data: obj })
-                        .on('click', () => {
-                          self.fullObject_ = obj;
-                          self.data = obj;
-                          self.isOpen_ = false;
+                        .addClass(section.disabled ? 'disabled' : '')
```

Comment:
`.enableClass('disabled', section.disabled)` would make more sense here. Not a big deal though.

Reply:
Fixed: #1791